tutorials/modules.html

```diff
@@ -0,0 +1,646 @@
+    <pre><code class="lang-js">
+// register the renderer before using it
+registerRenderer(autocompleteRenderer);
+    </code></pre>
+    <p>Now, you can use <code>autocompleteRenderer</code>, the full example looks like this:</p>
+    <pre><code class="lang-js">
+import Handsontable from 'handsontable/base';
+import { registerRenderer, autocompleteRenderer } from 'handsontable/renderers';
+registerRenderer(autocompleteRenderer);
+
+// use the renderer as you need
+new Handsontable(container, {
+  columns: [
+    {
+      renderer: 'autocomplete',
+    },
+  ],
+// rest of the settings
+});
+        </code></pre>
+        <p>And that is all! You can use the autocomplete renderer!</p>
+        </div>
+        <div class="example-container clearfix">
+          <h3 id="importing-validators">Importing validators</h3>
+          <p>
+            To use a specific validator you need to import it alongside the registering method. For instance, let's try adding the numeric validator.
+          </p>
+          <p>Start with importing the base, <code>numericValidator</code> and the <code>registerValidator</code> method.</p>
+            <pre><code class="lang-js">
+// remember to have the base imported
+import Handsontable from 'handsontable/base';
+
+// import the method for registering a validator
+// import the numericValidator
+import { registerValidator, numericValidator } from 'handsontable/validators';
+    </code></pre>
+    <p>Afterwards you need to use the <code>registerValidator</code> method to register <code>numericValidator</code>:</p>
+    <pre><code class="lang-js">
+// register the validator before using it
+registerValidator(numericValidator);
+    </code></pre>
+    <p>Now, you can use <code>numericValidator</code>, the full example looks like this:</p>
+    <pre><code class="lang-js">
+import Handsontable from 'handsontable/base';
+import { registerValidator, numericValidator } from 'handsontable/validators';
+registerValidator(numericValidator);
+
+// use the numeric validator where you need
+new Handsontable(container, {
+  columns: [
+    {
+      validator: 'numeric',
+    },
+  ]
+// rest of the settings
+});
+            </code></pre>
+            <p>And that is all! You can use the numeric validator!</p>
+        </div>
+        <div class="example-container clearfix">
+          <h3 id="importing-cell-types">Importing cell types</h3>
+              <p>
+                To use a specific cell type you need to import it alongside the registering method. Let's try adding the checkbox cell type.
+              </p>
+              <p>Start with importing the base, <code>CheckboxCellType</code> and the <code>registerCellType</code> method.</p>
+                <pre><code class="lang-js">
+// remember to have the base imported
+import Handsontable from 'handsontable/base';
+
+// import the method for registering a cell type
+// import the CheckboxCellType
+import { registerCellType, CheckboxCellType } from 'handsontable/cellTypes';
```

Comment:
Some examples have a new line, some doesn't. Dunno which formatting convention you wan to use
```suggestion
// remember to have the base imported
import Handsontable from 'handsontable/base';
// import the method for registering a cell type
// import the CheckboxCellType
import { registerCellType, CheckboxCellType } from 'handsontable/cellTypes';
```